### ISSUE TITLE
Joining the Unicode Consortium

### ISSUE THREAD

**Issue by eemeli** [openjs-foundation/standards#43]
Via `messageformat` I've become a participant in the [MessageFormat Working Group](https://github.com/unicode-org/message-format-wg/), which has now been chartered as a sub-WG of the Unicode CLDR group.

It would be nice if we could look into [joining the Unicode consortium](https://www.unicode.org/consortium/levels.html). I'm not sure how exactly it works, but it would appear possible and/or proper for us to become a liaison member, which would not carry a financial cost.

**Comment by eemeli** [comment]
Unicode blog post on the working group: http://blog.unicode.org/2020/01/new-unicode-working-group-message.html

**Comment by srl295** [comment]
@eemeli who's the right contact on the OpenJSF side?

**Comment by MylesBorins** [comment]
We discussed this in the standards team call today. Next steps are to bring this up in next weeks CPC meeting to get consensus on moving forward at which point it will go to the board to approve and then the foundation staff would execute on it.

TL;DR

1. CPC
2. Board
3. Staff executes

**Comment by srl295** [comment]
@MylesBorins you can contact me and i can route w/ any specific questions or contacts.

**Comment by kborchers** [comment]
Per our discussion in last week's CPC meeting, I have sent a message to the board asking for their input and will report back if there are any questions or guidance on how to proceed.

**Comment by kborchers** [comment]
Quick update - There has been support for this request expressed from a couple of board members and no objections. There is a request for legal to review the Unicode Consortium bylaws so I will follow up with that and if no issues are raised by the legal team, I imagine this will be approved to move forward soon.

**Comment by srl295** [comment]
> Quick update - There has been support for this request expressed from a couple of board members and no objections. There is a request for legal to review the Unicode Consortium bylaws so I will follow up with that and if no issues are raised by the legal team, I imagine this will be approved to move forward soon.

You can use the contact form https://home.unicode.org/connect/getting-involved/contact-unicode/ to connect to Unicode or write me and I can forward a message on.

**Comment by srl295** [comment]
@kborchers Also quick update: Unicode EMG has approved this in principle.

**Comment by kborchers** [comment]
Legal completed their review and have no concerns so we are free to proceed and @brianwarner is going to start working on the application.

**Comment by brianwarner** [comment]
Just a brief update, the liaison request has been submitted, we'll keep you posted!

**Comment by jorydotcom** [comment]
Per the last meeting, we understand we are still waiting to hear back from the consortium to approve our membership - is that correct @brianwarner

**Comment by brianwarner** [comment]
That's correct, yep.

**Comment by srl295** [comment]
OK. @brianwarner @eemeli etc… by the power vested in me (at this morning's EMG call… ) I got the action to communicate that this has been approved, and you are now a liaison member of Unicode.

Site updates, another email (Also from me) to follow!

Apology 👇  from me to follow:
Sorry that this took a while.

**Comment by brianwarner** [comment]
Alright! That's great news, thanks @srl295.

**Comment by srl295** [comment]
fyi @obensource

**Comment by jorydotcom** [comment]
this has been approved so we are closing the issue! thanks everyone!! @srl295 @eemeli

**Comment by obensource** [comment]
@srl295 @eemeli wahoo! Thanks all for you amazing work here! 🚀

Very excited to help develop this relationship with y'all. Here's to what's next! 🎉🍻🙌
